portfolio/src/main/webapp/index.html

```diff
@@ -42,9 +42,17 @@ <h1 style=font-size:40px class="heading-font">Apoorva Dhanala's Portfolio</h1>
       <h3 class="links-font"><a href="resume.html">MY RESUME</a></h3>
       <h3 class="links-font"><a href="photography.html">MY PHOTOGRAPHY</a></h3>
       <h3 class="links-font"><a href="favorites.html">FAVORITE SONGS, TV SHOWS, AND MORE!</a></h3>
```

Comment:
is links-font being used everywhere?

if so, consider just making it the default font for the body element

Reply:
No it's not - just for the 3 links at the bottom and one line in the comments box.